platform/core/src/services/ToolBarService/ToolBarService.js

```diff
@@ -17,24 +20,78 @@ export default class ToolBarService {
        */
     };
 
+    // TODO: Do we need to track per context? Or do we allow for a mixed
```

Comment:
That's a good point, you mean if you have 2 viewports that have different tools, yeah I think we'd need to. That can be for the future though.